src/lib/up-storage.js

```diff
@@ -418,6 +423,7 @@ class ProxyStorage implements IProxy {
     const readStream = this.request({
       uri_full: url,
       encoding: null,
+      strictSSL: this.strict_ssl,
```

Comment:
This is not necessary, you are setting `this.strict_ssl` already in the request method. Perhaps is a bit misleading the method is called as the import.